src/display/canvas.js

```diff
@@ -466,6 +459,7 @@ var CanvasGraphics = (function CanvasGraphicsClosure() {
     this.xobjs = null;
     this.commonObjs = commonObjs;
     this.objs = objs;
+    this.canvasFactory = canvasFactory || new DOMCanvasFactory();
```

Comment:
Based on https://github.com/mozilla/pdf.js/pull/8002#discussion_r99020974, this line should simply read
```javascript
this.canvasFactory = canvasFactory;
```